thunder/core/prims.py

```diff
@@ -2541,6 +2543,38 @@ def _div_numbers(a: Number, b: Number) -> Number:
     supported_input_dtypes=fp_math_dtypes,
 )
 
+#
+# Elementwise ternary prims
```

Comment:
:clap: 
This feels like a better category than what we had for `Conditional operations`, in which we have `where`.

Reply:
I overlooked `where`. Indeed, it can be in the same category as `lerp`!